app/src/main/java/org/wikipedia/page/PageActivity.java

```diff
@@ -1,926 +0,0 @@
-            pageFragment.updateBookmarkAndMenuOptionsFromDao();
-        }
-    };
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        app = (WikipediaApp) getApplicationContext();
-
-        PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
-
-        try {
-            setContentView(R.layout.activity_page);
-        } catch (Exception e) {
-            if ((!TextUtils.isEmpty(e.getMessage()) && e.getMessage().toLowerCase().contains("webview"))
-                    || (!TextUtils.isEmpty(ThrowableUtil.getInnermostThrowable(e).getMessage())
-                    && ThrowableUtil.getInnermostThrowable(e).getMessage().toLowerCase().contains("webview"))) {
-                // If the system failed to inflate our activity because of the WebView (which could
-                // be one of several types of exceptions), it likely means that the system WebView
-                // is in the process of being updated. In this case, show the user a message and
-                // bail immediately.
-                Toast.makeText(app, R.string.error_webview_updating, Toast.LENGTH_LONG).show();
-                finish();
-                return;
-            }
-            throw e;
-        }
-
-        unbinder = ButterKnife.bind(this);
-
-        disposables.add(app.getBus().subscribe(new EventBusConsumer()));
-
-        updateProgressBar(false);
-
-        pageFragment = (PageFragment) getSupportFragmentManager().findFragmentById(R.id.page_fragment);
-
-        setSupportActionBar(toolbar);
-        clearActionBarTitle();
-        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
-
-        searchButton.setOnClickListener(view -> startActivity(SearchActivity.newIntent(PageActivity.this, TOOLBAR, null)));
-
-        tabsButton.setColor(ResourceUtil.getThemedColor(this, R.attr.material_theme_de_emphasised_color));
-        FeedbackUtil.setButtonLongPressToast(tabsButton, overflowButton);
-        tabsButton.updateTabCount(false);
-
-        toolbarHideHandler = new ViewHideHandler(toolbarContainerView, null, Gravity.TOP);
-
-        drawerLayout.setScrimColor(Color.TRANSPARENT);
-        containerWithNavTrigger.setCallback(this);
-
-        ViewCompat.setOnApplyWindowInsetsListener(drawerLayout, (v, insets) -> {
-            toolbarContainerView.setPadding(0, insets.getSystemWindowInsetTop(), 0, 0);
-            pageFragment.updateInsets(insets);
-            return insets;
-        });
-
-        hasTransitionAnimation = getIntent().getBooleanExtra(Constants.INTENT_EXTRA_HAS_TRANSITION_ANIM, false);
-        wikiArticleCardView.setVisibility(hasTransitionAnimation ? View.VISIBLE : View.GONE);
-
-        boolean languageChanged = false;
-        if (savedInstanceState != null) {
-            if (savedInstanceState.getBoolean("isSearching")) {
-                startActivity(SearchActivity.newIntent(this, TOOLBAR, null));
-            }
```

Comment:
This will be removed since there's no other place sending "isSearching" to the bundle.